### PR TITLE
add rtl support

### PR DESCRIPTION
**Pull request by ariaieboy**
image

with selected property
image

### CODE REVIEW

**Comment by ariaieboy** [comment]
@CodeWithDennis it's ready to merge

**Comment by CodeWithDennis** [comment]
@ariaieboy If you can take a look at my latest comments and fix them we can merge it =)

**Comment by ariaieboy** [comment]
@CodeWithDennis I can't find your comments.

**Comment by CodeWithDennis** [comment]
> @CodeWithDennis I can't find your comments.

https://github.com/CodeWithDennis/filament-select-tree/pull/45/files

File: resources/css/custom.css [hunk 1]
```diff
@@ -246,3 +246,11 @@ html.dark .treeselect--disabled {
 html.dark .treeselect-list__item--disabled .treeselect-list__item-checkbox-container {
     background-color: hsl(0deg 0% 30.77% / 5%);
 }
+[dir='rtl'] .treeselect-input__operators {
```
Comment (ariaieboy) [inline resources/css/custom.css:249]: test

**Comment by ariaieboy** [comment]
> > @CodeWithDennis I can't find your comments.
> 
> https://github.com/CodeWithDennis/filament-select-tree/pull/45/files

I can't see any comments on the codes I created a test comment.

**Comment by CodeWithDennis** [comment]
> > > @CodeWithDennis I can't find your comments.
> > 
> > 
> > https://github.com/CodeWithDennis/filament-select-tree/pull/45/files
> 
> I can't see any comments on the codes I created a test comment.

TLDR:

Can we make this prettier:

1. 
```
rtl: '{{ ((__('filament-panels::layout.direction')??'ltr')=='rtl'?true:false) }}'
```

2. src/SelectTree.php line 337 you removed a space

**Comment by ariaieboy** [comment]
@CodeWithDennis Is it good now?
